### PR TITLE
Update accepted chrome versions (48+) for chrome os

### PR DESCRIPTION
**Pull request by eacraft13**
Fix for https://oat-sa.atlassian.net/browse/TAO-3627

Note: it was asked to accept versions 45+ of firefox. I'm getting clarification on this right now if this was meant for Chrome OS (bc, as I currently understand it, chrome is the only acceptable browser on chrome os).

### CODE REVIEW

**Comment by antoinerobin** [comment]
this will work for chrome without any issue. You have to add a line for firefox.
Please update the version number in the manifest and the updater

File: include/compatibility.json [hunk 1]
```diff
@@ -1,51 +1,56 @@
...
+    {"compatible" : 1, "os" : "Windows", "osVersion" : "8", "browser" : "Firefox", "versions" : [35]},
+    {"compatible" : 1, "os" : "Windows", "osVersion" : "8", "browser" : "Chrome", "versions" : [39]},
+    {"compatible" : 1, "os" : "Windows", "osVersion" : "8", "browser" : "Internet Explorer", "versions" : [10]},
 
-    {"compatible" : 1, "os" : "Windows", "osVersion" : "7", "browser" : "Firefox", "versions" : [34,35,36, 37, 38, 39, 40, 41, 42, 43, 44, 45, 50]},
-    {"compatible" : 1, "os" : "Windows", "osVersion" : "7", "browser" : "Internet Explorer", "versions" : [10, 11]},
-    {"compatible" : 1, "os" : "Windows", "osVersion" : "7", "browser" : "Chrome", "versions" : [40, 41, 42, 43, 44, 45, 46, 47, 48, 49, 55]},
+    {"compatible" : 1, "os" : "Windows", "osVersion" : "7", "browser" : "Firefox", "versions" : [34]},
+    {"compatible" : 1, "os" : "Windows", "osVersion" : "7", "browser" : "Chrome", "versions" : [40]},
+    {"compatible" : 1, "os" : "Windows", "osVersion" : "7", "browser" : "Internet Explorer", "versions" : [10]},
```
Comment (antoinerobin) [inline include/compatibility.json]: it will only allow IE 10 not the 11

File: include/compatibility.json [hunk 2]
```diff
@@ -1,51 +1,56 @@
...
-    {"compatible" : 1, "os" : "Windows", "osVersion" : "8", "browser" : "Internet Explorer", "versions" : [10,11]},
+    {"compatible" : 1, "os" : "Windows", "osVersion" : "8", "browser" : "Firefox", "versions" : [35]},
+    {"compatible" : 1, "os" : "Windows", "osVersion" : "8", "browser" : "Chrome", "versions" : [39]},
+    {"compatible" : 1, "os" : "Windows", "osVersion" : "8", "browser" : "Internet Explorer", "versions" : [10]},
 
-    {"compatible" : 1, "os" : "Windows", "osVersion" : "7", "browser" : "Firefox", "versions" : [34,35,36, 37, 38, 39, 40, 41, 42, 43, 44, 45, 50]},
-    {"compatible" : 1, "os" : "Windows", "osVersion" : "7", "browser" : "Internet Explorer", "versions" : [10, 11]},
-    {"compatible" : 1, "os" : "Windows", "osVersion" : "7", "browser" : "Chrome", "versions" : [40, 41, 42, 43, 44, 45, 46, 47, 48, 49, 55]},
+    {"compatible" : 1, "os" : "Windows", "osVersion" : "7", "browser" : "Firefox", "versions" : [34]},
+    {"compatible" : 1, "os" : "Windows", "osVersion" : "7", "browser" : "Chrome", "versions" : [40]},
```
Comment (antoinerobin) [inline include/compatibility.json]: so we assume that firefox 45-50 and chrome 49-55 are tested ?

Comment (eacraft13) [inline include/compatibility.json]: This is to allow for them to be tested. This question should be readdressed after qa has been able to test the different environments.

File: include/compatibility.json (hunk 1 above)
Comment (eacraft13) [inline include/compatibility.json]: This was incorrect. I will modify the code to support multiple versions of IE.

**Pull request merged**